config/config_test.go

```diff
@@ -2034,3 +2036,42 @@ func TestRemoveExchange(t *testing.T) {
 		t.Fatal("exchange shouldn't exist")
 	}
 }
+
+func TestConfig_GetDataPath(t *testing.T) {
+	tests := []struct {
+		name string
+		dir  string
+		elem []string
+		want string
+	}{
+		{
+			name: "empty",
+			dir:  "",
+			elem: []string{},
+			want: common.GetDefaultDataDir(runtime.GOOS),
+		},
+		{
+			name: "empty a b",
+			dir:  "",
+			elem: []string{"a", "b"},
+			want: filepath.Join(common.GetDefaultDataDir(runtime.GOOS), "a", "b"),
+		},
+		{
+			name: "target",
+			dir:  "target",
+			elem: []string{"a", "b"},
+			want: filepath.Join("target", "a", "b"),
+		},
+	}
+	for _, tt := range tests {
+		tt := tt
```

Comment:
need to add t.Parallel() otherwise this line doesnt really help. without t.Parallel() normal iteration should suffice